src/Stripe.net/Services/SubscriptionSchedules/SubscriptionScheduleRenewalIntervalOptions.cs

```diff
@@ -0,0 +1,22 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class SubscriptionScheduleRenewalIntervalOptions : StripeEntity
+    {
+        /// <summary>
+        /// Interval at which to renew the subscription schedule for when it ends. Possible values
+        /// are <code>day</code>, <code>week</code>, <code>month</code>, or <code>year</code>.
+        /// </summary>
+        // TODO: Decide if using PlanInterval
```

Comment:
Should I be using `PlanInterval` here? It's the same list, but also feels like a different meaning (and maybe we would like this to be renamed to `BillingInterval`?)

Reply:
Yeah, it's annoying, and the OpenAPI spec is no help here either as the enums are not defined as a single schema that is referenced from different parameters, but rather duplicated for each parameter.

I'd leave as is for now, but it's something we should fix upstream.